plugins/arDominionB5Plugin/scss/_layout.scss

```diff
@@ -34,7 +34,7 @@ body {
 
 .dropdown-menu {
   box-shadow: $box-shadow;
-  
+
```

Comment:
Ups ... thanks!

Reply:
Added this in a different commit, please rebase ;)

Reply:
👍